script/all-utilities

```diff
@@ -149,13 +149,22 @@ function retry() {
 # Download api Keys from Cloud storage and source the file.
 function set_api_keys() {
   local api_key_directory="$(mktemp -d)"
-  gsutil cp gs://client-secret-files/api_keys \
+  $GSUTIL cp gs://client-secret-files/api_keys \
         "${api_key_directory}/api_keys" \
           || error_exit "Failed to download API key file."
 
   source "${api_key_directory}/api_keys"
 }
 
+# Download test-client keys from Cloud storage
+function get_test_client_key() {
+  local key_path=$1
```

Comment:
It is possible but I'd like to keep the key in the working directory, so it won't be downloaded again.